interview-questions.md

```diff
@@ -9,59 +9,59 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: when we have two+ vars that happen to have the same name, we use 'this' when we are referring to the global variable or the argument
 
-  Researched answer:
+  Researched answer: In most cases, the value of this is determined by how a function is called (runtime binding).
 
 
 
 2. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: yarn allows us to run a react and test file when coding. A node_modules is added when you run yarn
 
-  Researched answer:
+  Researched answer: Yarn is a package manager for your code
```

Comment:
Can you expand on yarn: What other files does it add? How does yarn work?

Reply:
yarn adds dependencies files. When you add yarn, node_modules is added and this contains the files we need for react